packages/kbn-utility-types/README.md

```diff
@@ -19,6 +19,7 @@ type B = UnwrapPromise<A>; // string
 ## Reference
 
 - `Ensure<T, X>` &mdash; Makes sure `T` is of type `X`.
+- `Mutable<T>` &mdash; Removes `readonly` attribute from all properties of `T`.
```

Comment:
actually, this doesn't seem used in this PR? Is it needed?

Reply:
I was using it in this PR. But after some code changes stopped using it. I'll remove it from this PR.